### ISSUE TITLE
Register extension fails if .blend not already associated with Blender (Windows 10)

### ISSUE THREAD

**Issue by dreamparacite** [DotBow/Blender-Launcher#34]
I am using Windows 10 Pro Version 10.0.19041 Build 19041. At first I thought register extension didn't work because my .blend files were "blank" and not associated with any program. Here's what my testing has found:

- If .blend file type is not associated with any program, doing "register extension" does nothing. 
- If .blend file type is associated with another unrelated program, register extension doesn't work.
- If the .blend file type is associated with some version of Blender, then register extension does work. 

So basically, if windows already associates .blend with Blender, register extension will change what version is opens with. If it is not associated with any program, or an unrelated program, register extension won't fix it.

**Comment by DotBow** [comment]
Hi!

> If .blend file type is not associated with any program, doing "register extension" does nothing.

Can't reproduce because it seems deleting extension association in Windows is so hard that doesn't even work through regedit...
Let me know if there is a solution for this one.
However, I never had issues with it before on fresh install of Windows.

> If .blend file type is associated with another unrelated program, register extension doesn't work.

It does what it should, the problem is that Windows then associate it with two programs rather than replacing existing with new one.
Right now BL uses Blender internal method for registering extension (blender.exe -r), so it seems like the Blender and Windows issue.
Furthermore, it is a very weird case when .blend associated with unrelated program, it can be done only if user (like us 😄) want to break everything.

It still 'can' be done via Python and winreg, but would be super painful. I think it is better to report a bug in Blender.

**Comment by dreamparacite** [comment]
Thanks for the response! It is hard to remove program associations, haven't found a good way to do that yet, was trying to figure that out myself to reproduce it. Also it's definitely not likely users will associate .blend with unrelated programs unless intentional, like you said. I'll just make sure .blend is associated with Blender first, as that works as expected. Thanks for clarifying!

**Comment by DotBow** [comment]
Thanks for reporting an issue!
I will reopen issue because this one still bothers me:
> I'll just make sure .blend is associated with Blender first, as that works as expected
At least it can be written as note in wiki.

**Comment by DotBow** [comment]
Added notes to documentation
